bootloader/ota-dfu-python/README.md

```diff
@@ -1,13 +1,13 @@
 # Python nRF5 OTA DFU Controller
 
 So... this is my fork of dingara's fork of astronomer80's fork of
-foldedtoad's Python OTA DFU utility. 
+foldedtoad's Python OTA DFU utility.
 
-My own contribution is little more than a brute force conversion to 
-python3. It is sparsely tested so there are likely to be a few 
+My own contribution is little more than a brute force conversion to
```

Comment:
I've always kept to the "one sentence one line" rule for markdown. Makes diffs MUCH more readable and formatting won't mess with a small change

Reply:
I'm inclined to agree, but none of the formatters I tested so far support this. We'll have to discuss this later. This PR isn't intended to fix everything, but only the most obvious issues.